api/institutions/views.py

```diff
@@ -418,10 +416,22 @@ class InstitutionImpactList(JSONAPIBaseView, ListFilterMixin, generics.ListAPIVi
 
     view_category = 'institutions'
 
+    csv_export = False
+
+    def initiate_csv_export_settings(self):
+        self.pagination_class = MaxSizePagination
+        self.csv_export = True
+
+    def is_csv_export(self):
+        raise NotImplementedError()
```

Comment:
what if `is_csv_export` were defined here (using the base `MetricsCSVRenderer`) instead of in each child class?
```suggestion
    def is_csv_export(self):
        return isinstance(self.request.accepted_renderer, MetricsCSVRenderer)
```
seems like could then get rid of `self.csv_export` and simplify the children

Reply:
Using the base `MetricsCSVRenderer` makes sense and should work! I completely missed that possibility - thanks.